external-workshops/course-website.md

```diff
@@ -0,0 +1,23 @@
+# Creating a course website
+
+We strongly recommend creating a course website for your workshop. 
+Course websites ([example](https://alexslemonade.github.io/2023-june-training/)) are a centralized place for attendees to find information necessary for participating in the workshop. 
+At the Data Lab, we create a website for each course using our [`training-specific-template`](https://github.com/AlexsLemonade/training-specific-template) repository, which you may use as a template for your own website.
+To do this, you must create a GitHub account if you do not already have one. 
+
+**1.)** First, create your own GitHub repository from `training-specific-template`, [using GitHub’s instructions for creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template). 
+<img src="screenshots/training-specific-new-template.png" width=800>
+
+**2.)** Make sure you select your own GitHub account as the owner of the new repository and choose to make your repository public, so your participants will have access. 
```

Comment:
This is not actually necessary - see https://github.com/AlexsLemonade/training-specific-template/pull/157

I might just tell them to refer to the template repo's instructions at this point - either way in general, we should make sure to say that that repo _has_ instructions for them too!

Reply:
✅